.github/release_template.md

```diff
@@ -0,0 +1,50 @@
+<!--
+This is the template to release a new version on the App Store
+-->
+
+Release a new version
+## TASKS:
+
+Git & Code
+
+- [ ] [GIT] Create branch `release/[major].[minor].[patch]` (freeze the code)
+- [ ] [DEV] Update version number `[major].[minor].[patch]`
+- [ ] [DIS] Update [changelog](https://github.com/owncloud/ios-app/blob/master/CHANGELOG.md)
+- [ ] [QA] Design Test plan
+- [ ] [QA] Regression Test plan
+- [ ] [DOC] Update user manual with the new functionalities
+
+App Store
+
+- [ ] [DIS] iTunes Connect: Create a new version following the `[major].[minor].[patch]`
+- [ ] [DIS] iTunes Connect: Update screenshots if needed
+- [ ] [DIS] Upload the binary to the App Store
+- [ ] [DIS] iTunes Connect: When release (manually, automatically, automatically not early than date)
+- [ ] [DIS] iTunes Connect: How to releae (immediately to all the users o 7-day period phased release)
+- [ ] [DIS] iTunes Connect: Reset iOS Summary Rating (Keep existing or reset the rating)
```

Comment:
Please make the proposed change, in your revised version:

```
- [ ] [DIS] iTunes Connect: Time of release (manually / automatically / not before date)
- [ ] [DIS] iTunes Connect: Availability (to all users / staged rollout over 7 day period)
- [ ] [DIS] iTunes Connect: Reviews (Keep existing / Delete existing)
```

Reply:
:+1: